src/main/java/wolox/training/models/Book.java

```diff
@@ -152,6 +160,7 @@ public String getYear() {
     }
 
     public void setYear(String year) {
+        Preconditions.checkArgument(Strings.isNullOrEmpty(year), "Year field is required");
```

Comment:
If you want to make this better, you can check the year is not greater than the actual